webapp/packages/plugin-connections-administration/src/Administration/Connections/ConnectionsTable/ConnectionsTable.tsx

```diff
@@ -42,6 +43,15 @@ export const ConnectionsTable = observer<Props>(function ConnectionsTable({
   expandedItems,
 }) {
   const translate = useTranslate();
+  const projectService = useService(ProjectsService);
+  const projectsLoader = useResource(ConnectionsTable, ProjectInfoResource, CachedMapAllKey);
```

Comment:
projectsLoader -> projectInfoResource

Reply:
it's not exactly resource, it's resource's data loader